core/src/cluster_info.rs

```diff
@@ -2171,13 +2180,22 @@ impl ClusterInfo {
         }
 
         let (stakes, epoch_time_ms) = Self::get_stakes_and_epoch_time(bank_forks);
-
+        let feature_set = bank_forks.map(|bank_forks| {
+            bank_forks
+                .read()
+                .unwrap()
+                .working_bank()
```

Comment:
It'll be better to use `.root_bank()`.  With the `.working_bank()`, we could see a feature get enabled and then disabled again (if the feature happens to get enabled in a minority fork).    Otherwise lgtm!

Reply:
done.

There are other places in this file which are using `.working_bank` (e.g. for stakes). Do you think they should also use `.root_bank` instead?

Reply:
I'd say so, but only for consistency.  It looks like it shouldn't matter in practice either way though